R/checkMeasures.R

```diff
@@ -1,8 +1,11 @@
-checkMeasures = function(measures, obj, aggr = NULL) {
+checkMeasures = function(measures, obj, aggr = NULL, na.rm) {
   if (missing(measures)) {
     measures = list(getDefaultMeasure(obj))
   } else {
     measures = ensureVector(measures, n = 1L, cl = "Measure")
+    for (i in seq_along(measures)) {
+      measures[[i]]$aggr$na.rm = na.rm
+    }
```

Comment:
Document that calling `checkMeasures` sets `na.rm` for all the measures please. And there should be a default value of `FALSE` again.